script/Arcueid/Day 10/10_00_ARC10_7.txt

```diff
@@ -16,12 +16,13 @@ Besides, you couldn't kill that kid with your skill. You'd be the one killed ins
 [sha:12c35238b462c193b2ac60577440ba9fbb8b9ce7]{
 -- Page 756, Offset 26324.
 -- 「なんでって、そろそろ<調律|ピアノ>の影響が出る頃だからだよ。
+// double check: this is correct literally, but I'm not sure what the fuck it means
```

Comment:
I'm pretty sure this is right (this is referring to Mario's power I think?) iirc its called Piano Machine or something like that